### PR TITLE
R4R: Improve CLI and LCD APIs 

### PR DESCRIPTION
**Pull request by chengwenxi**
Resolves: https://github.com/irisnet/irishub/issues/1245

### CODE REVIEW

File: client/bank/lcd/rest.go [hunk 1]
```diff
@@ -13,14 +13,11 @@ func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec)
 		QueryAccountRequestHandlerFn(cdc, utils.GetAccountDecoder(cdc), cliCtx)).Methods("GET")
 	r.HandleFunc("/bank/balances/{address}",
 		QueryBalancesRequestHandlerFn(cdc, utils.GetAccountDecoder(cdc), cliCtx)).Methods("GET")
-	r.HandleFunc("/bank/coin/{coin-type}",
+	r.HandleFunc("/bank/coins/{type}",
 		QueryCoinTypeRequestHandlerFn(cdc, cliCtx)).Methods("GET")
```
Comment (haifengxi) [inline client/bank/lcd/rest.go]: /bank/coins/{type} --> /bank/assets/{assetId}/type

File: client/bank/lcd/rest.go [hunk 2]
```diff
@@ -13,14 +13,11 @@ func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec)
...
 	r.HandleFunc("/bank/token-stats",
 		QueryTokenStatsRequestHandlerFn(cdc, cliCtx)).Methods("GET")
```
Comment (haifengxi) [inline client/bank/lcd/rest.go]: /bank/token-stats --> /bank/assets/{assetId}/statistics

File: client/distribution/lcd/rest.go [hunk 3]
```diff
@@ -13,7 +13,7 @@ func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec)
 
 	r.HandleFunc("/distribution/{delegatorAddr}/withdrawAddress",
 		QueryWithdrawAddressHandlerFn(cliCtx)).Methods("GET")
-	r.HandleFunc("/distribution/{delegatorAddr}/distrInfo/{validatorAddr}",
+	r.HandleFunc("/distribution/{delegatorAddr}/distrInfos/{validatorAddr}",
 		QueryDelegationDistInfoHandlerFn(cliCtx)).Methods("GET")
```
Comment (haifengxi) [inline client/distribution/lcd/rest.go]: distrInfos --> distrInfo, as `information` is uncountable noun

File: client/bank/lcd/rest.go [hunk 4]
```diff
@@ -13,14 +13,11 @@ func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec)
 		QueryAccountRequestHandlerFn(cdc, utils.GetAccountDecoder(cdc), cliCtx)).Methods("GET")
 	r.HandleFunc("/bank/balances/{address}",
 		QueryBalancesRequestHandlerFn(cdc, utils.GetAccountDecoder(cdc), cliCtx)).Methods("GET")
```
Comment (haifengxi) [inline client/bank/lcd/rest.go]: /bank/balances/{address} --> /bank/accounts/{address}

File: docs/light-client/README.md [hunk 5]
```diff
@@ -66,39 +66,40 @@ Once IRISLCD is started, you can open `localhost:1317/swagger-ui/` in your explo
     7. `GET /auth/accounts/{address}`: Get the account information on blockchain
```
Comment (haifengxi) [inline docs/light-client/README.md]: /auth/accounts/{address} --> /bank/accounts/{address}

File: docs/zh/light-client/README.md [hunk 6]
```diff
@@ -67,39 +67,40 @@ IRISLCD启动以后，您可以在浏览器中访问`localhost:1317/swagger-ui/`
     7. `GET /auth/accounts/{address}`: 查询秘钥对象账户的信息
     8. `POST /keys/{name}/recover`: 通过seed恢复一个账户
```
Comment (haifengxi) [inline docs/zh/light-client/README.md]: /auth/accounts/{address} --> /bank/accounts/{address}

**Comment by haifengxi** [comment]
Let's review the above comments in a face-to-face meeting @chengwenxi

File: client/bank/lcd/rest.go (hunk 4 above)
Comment (chengwenxi) [reply client/bank/lcd/rest.go]: remove

File: client/distribution/lcd/rest.go [hunk 7]
```diff
@@ -13,7 +13,7 @@ func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec)
...
 	r.HandleFunc("/distribution/{delegatorAddr}/distrInfos",
```
Comment (chengwenxi) [inline client/distribution/lcd/rest.go]: distrInfos --> distrInfo

File: docs/light-client/README.md (hunk 5 above)
Comment (chengwenxi) [reply docs/light-client/README.md]: The URL is /auth/accounts/{address}  in the code, need to be modified?

Comment (haifengxi) [reply docs/light-client/README.md]: Yes, please do.

File: docs/zh/light-client/README.md [hunk 8]
```diff
@@ -75,9 +75,8 @@ IRISLCD启动以后，您可以在浏览器中访问`localhost:1317/swagger-ui/`
 4. Bank模块的APIs
     1. `GET /bank/coins/{coin-type}`: 查询coin的类型信息
     2. `GET /bank/token-stats`: 查询token统计信息
-    3. `GET /bank/balances/{address}`: 查询账户的token数量
-    4. `POST /bank/accounts/{address}/transfers`: 发起转账交易
-    5. `POST /bank/burn`: 销毁token
+    3. `POST /bank/accounts/{address}/transfers`: 发起转账交易
+    4. `POST /bank/burn`: 销毁token
```
Comment (haifengxi) [inline docs/zh/light-client/README.md]: /bank/burn --> /bank/accounts/{address}/burn

File: client/bank/lcd/rest.go [hunk 9]
```diff
@@ -17,5 +17,5 @@ func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec)
 		QueryTokenStatsRequestHandlerFn(cdc, cliCtx)).Methods("GET")
 
 	r.HandleFunc("/bank/accounts/{address}/transfers", SendRequestHandlerFn(cdc, cliCtx)).Methods("POST")
```
Comment (haifengxi) [inline client/bank/lcd/rest.go]: /bank/accounts/{address}/transfers --> /bank/accounts/{address}/send

Comment (chengwenxi) [reply client/bank/lcd/rest.go]: A send transaction will add a transfers resouce, so I think it's reasonable.

**Pull request merged**